aw/Extensions/Reach.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import MapKit
+
+extension Reach {
+
+    var readingFormatted: String {
+        guard let lastGageReading = lastGageReading, let unit = unit else {
+            //print(self.lastGageReading, self.unit)
+            return "n/a"
+        }
+        return "\(lastGageReading) \(unit)"
+    }
+
+    var color: UIColor {
+        let con = Condition.fromApi(condition: condition ?? "")
+
+        return con.color
+    }
+
+    var icon: UIImage? {
+        return Condition.fromApi(condition: condition ?? "").icon
+    }
+
+    var sectionCleanedHTML: String? {
```

Comment:
You probably don't need to indicate "cleaned" or "formatted." That's implied as part of the responsibility of the view, to do any display related data transformations. On the other hand, if you're doing a specific transformation that involves some calculation and some complexity, you should call that out or split into another method.

Reply:
Ya, the name sucks. I tried to push common property display styles / formatting related to the CoreData entities into their extensions so that they were easier to reuse on different views.